build/buildx/drivers/remote.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Remote driver"
```

Comment:
I'm happy with the note at the beginning noting this as a pre-release feature - but I think we should definitely try and make sure we don't do this a lot.

Could we maybe consider just dropping this page for now? Although, I'm not sure how we'd modify the index to drop it from the list.

Reply:
Sure we can drop it, lgtm

Reply:
Argh if we drop this page, the `index.md` is not aligned :sweat_smile:

Reply:
Let's keep it for now, I will change the `info` note to a warning one.

Reply:
Sounds good, it should get resolved quite soon, in the next bulidx release.